packages/react-tinacms-editor/src/tinacms-inline/inline-wysiwyg.tsx

```diff
@@ -51,13 +51,20 @@ export function InlineWysiwyg({
     <InlineField name={name}>
       {({ input, form }: InlineWysiwygRenderProps) => {
         return (
-          <FocusRing name={name} options={focusRing}>
+          <FocusRing name={input.name} options={focusRing}>
```

Comment:
```suggestion
          /**
           * Note: We use `input.name` not `name` here because
           * the given name is only relative to the block, not
           * the absolute path in the form.
           */
          <FocusRing name={input.name} options={focusRing}>
```